plugins/azure/virtualmachinescaleset/scaleSetLinuxSSHEnabled.js

```diff
@@ -0,0 +1,56 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'VM Scale Set Linux SSH Enabled',
+    category: 'Virtual Machine Scale Set',
+    domain: 'Compute',
+    description: 'Ensures that Azure Virtual Machine scale sets with Linux OS has SSH enabled.',
+    more_info: 'SSH is an encrypted connection protocol that allows secure sign-ins over unsecured connections. SSH is the default connection protocol for Linux VMs hosted in Azure.',
+    link: 'https://learn.microsoft.com/en-us/azure/virtual-machines/linux/ssh-from-windows',
+    recommended_action: 'Remove existing scale set and create new with SSH enabled',
+    apis: ['virtualMachineScaleSets:listAll'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var locations = helpers.locations(settings.govcloud);
+
+        async.each(locations.vmScaleSet, function(location, rcb) {
```

Comment:
We have two resources in locations.js vmScaleSet and virtualMachineScaleSets, can you use virtualMachineScaleSets instead making it consistent, and also remove the other from locations file along with its usage